src/pivot-table/hooks/use-column-width.ts

```diff
@@ -166,7 +179,7 @@ export default function useColumnWidth(
   );
 
   const leftGridWidth = useMemo(
-    () => leftGridColumnWidths.reduce((totalWidth, w) => totalWidth + w, 0),
+    () => leftGridColumnWidths.modifiedColumnWidth.reduce((totalWidth, w) => totalWidth + w, 0),
```

Comment:
If I understand what you are trying to do, the only changes you need is to scrap the whole scaling logic in `leftGridColumnWidths` and only return one array as before. And then here you do
```suggestion
    () => Math.min(rect.width * LEFT_GRID_MAX_WIDTH_RATIO, leftGridColumnWidths.reduce((totalWidth, w) => totalWidth + w, 0)),
```

Reply:
Seems like that would work to me

Reply:
I actually want to escape runtime/more calculations on each call to `leftGridWidth`,
by latest changes, it's calculated once within the `leftGridWidthInfo` and `useMemo`-ised